src/lib.rs

```diff
@@ -18,9 +18,11 @@ pub fn create_loader_runtime() -> Result<tokio::runtime::Runtime, Error> {
         .context("Error creating loader tokio runtime")
 }
 
-pub fn start_logger() {
+pub fn start_logger() -> Result<(), Error> {
     let env = Env::new().filter_or("RUST_LOG", "INFO");
-    env_logger::init_from_env(env);
+    env_logger::try_init_from_env(env)
```

Comment:
Just curious : In which case had you the error ?

Reply:
In the integration test. I added log to have a best feedback on what's gone wrong.

Reply:
as we launch that multiple time, it fail after the first time we call it.

Reply:
Ok for this PR we should change this behavior in another PR :+1: